docs/03-TechnicalInfrastructure.md

```diff
@@ -90,31 +94,35 @@ Jiv images produced by the imaging insertion pipeline are organized per
 
 #### The `logs` directory
 
-The logs of the scripts are created under `data/logs` in `/data/project`.
+The logs of the scripts are created under `data/logs` in `/data/$PROJECT`.
     
 ```
-## Content of the /data/project/data/logs directory
+## Content of the /data/$PROJECT/data/logs directory
 .
-|__ TarLoad.log
-    |__ DTIPrep_pipeline
-    |   |__ DTI_QC`date`.log
-    |   |__ DTI_QC`date`.log
-    |__ DTIPrep_register
-    |   |__ DTIregister`date`.log
-    |   |__ DTIregister`date`.log
-    |__ registerProcessed
-        |__ registerProcessed`date`.log
-        |__ registerProcessed`date`.log
+|__ TarLoad-`hh-mm-xxxxxx`.log
+|__ DTIPrep_pipeline*
+|   |__ DTI_QC`date`.log
```

Comment:
@cmadjar 
the `date` thing is not being formatted properly because it is already in a code section!
not sure if it worth investigating this, but I can not seem to figure out how!